resources/license-header.js

```diff
@@ -17,4 +17,4 @@
  * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
  * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
- */
\ No newline at end of file
+ */
```

Comment:
That could be causing the license header issue you were having

Reply:
Yep - looks like that was it!